src/wx/config/internal/option-internal.cpp

```diff
@@ -328,9 +328,10 @@ std::array<Option, kNbOptions>& Option::All() {
         Option(OptionID::kPrefSkipSaveGameCheats, &coreOptions.skipSaveGameCheats, 0, 1),
         Option(OptionID::kPrefSkipSaveGameBattery, &coreOptions.skipSaveGameBattery, 0, 1),
         Option(OptionID::kPrefThrottle, &coreOptions.throttle, 0, 450),
-        Option(OptionID::kPrefSpeedupThrottle, &coreOptions.speedup_throttle, 0, 3000),
-        Option(OptionID::kPrefSpeedupFrameSkip, &coreOptions.speedup_frame_skip, 0, 300),
+        Option(OptionID::kPrefSpeedupThrottle, &coreOptions.speedup_throttle, 0, 450),
+        Option(OptionID::kPrefSpeedupFrameSkip, &coreOptions.speedup_frame_skip, 0, 40),
```

Comment:
This is minor and I doubt anyone is going to be hit by this but since you're lowering the higher bound here, could you also silently force-update the value if it's above the higher bound here: https://github.com/visualboyadvance-m/visualboyadvance-m/blob/7a0826a60ccec45bfdd0d92070bb1d8fb783a0e6/src/wx/opts.cpp#L255?
We need a better way of handling this but this will do for now.